Cargo.lock

```diff
@@ -1342,7 +1342,7 @@ dependencies = [
  "import_map",
  "indexmap",
  "log",
- "monch",
+ "monch 0.4.3",
```

Comment:
This duplicate monch dep will be fixed in https://github.com/denoland/deno_graph/pull/361